services/ingredient.js

```diff
@@ -47,10 +47,13 @@ const getIngredientByName = name => getDbConn(dbAddr).any(
 const getRecipeIngredients = id => getDbConn(dbAddr).any(
```

Comment:
another naming recommendation. This should ideally be the `recipeId` instead of `id` since you're querying by `recipe_id`

Reply:
Perfect, will change it!